### ISSUE TITLE
Update Hugeicons set to add missing icons

### ISSUE THREAD

**Issue by buhodev** [iconify/icon-sets#150]
There are some icons missing, for example, but not limited to:

- home-07
- home-08
- home-09
- home-10
- home-11
- home-12
- home-13

**Comment by buhodev** [comment]
It looks like there is an [issue](https://github.com/hugeicons/hugeicons-react/issues/16) tracking this with no response so far

**Comment by cyberalien** [comment]
Yes, there is also a package for SVG files, it is even further out of date than React component. Looks like icon set is not maintained by developers.

**Comment by cyberalien** [comment]
Updated, thanks to @anhthang

See https://github.com/iconify/icon-sets/issues/184
